package.json

```diff
@@ -35,6 +35,19 @@
     "test": "jest",
     "types": "tsc --noEmit"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged && npm run test"
```

Comment:
My only question here would be if we actually want to run tests on every commit. My worry is that it'll slow down the development process, since Jest does take quite a while (a few seconds) to start and run.

Given that we have a CI/CD pipeline that runs tests, do you think this is worth doing?